src/app/+step1/step1.component.html

```diff
@@ -1,9 +1,19 @@
 <p>This is step1.</p>
 
   <form [ngFormModel]="step1">
-    <form-field [field]="fm.getField('input_name')"></form-field>
+    <form-field [field]="fm.getField('input_name')" 
+     (value)="update($event.field, $event.value)"></form-field>
```

Comment:
for those wondering why do we need to have (value) on every field, its so we can control the out put on custom actions like for region field